src/BasisTheory.net/Common/Utilities/UserAgentUtility.cs

```diff
@@ -0,0 +1,66 @@
+using System;
+using System.Reflection;
+using System.Collections.Generic;
+using System.Linq;
+using BasisTheory.net.Common.Entities;
+using static System.Runtime.InteropServices.RuntimeInformation;
+
+namespace BasisTheory.net.Common.Utilities
+{
+    public static class UserAgentUtility
+    {
+        static UserAgentUtility()
+        {
+            Client = "BasisTheory.net";
+            ClientVersion = Assembly.GetAssembly(typeof(UserAgentUtility)).GetName().Version.ToString();
```

Comment:
`Version.ToSTring()` by default returns 4 segments. Do we want to override to use `.ToString(3)` to only return `<major>.<minor>.<patch>` versions?

Reply:
How are we dynamically setting the assembly version when we publish the Nuget package? Pulling the assembly version in an existing version of BasisTheory.net always returns `1.0.0.0`.